plugin/ensime.vim

```diff
@@ -61,8 +61,10 @@ augroup ensime
     autocmd VimLeave *.java,*.scala call ensime#au_vim_leave(expand("<afile>"))
     autocmd VimEnter *.java,*.scala call ensime#au_vim_enter(expand("<afile>"))
     autocmd BufLeave *.java,*.scala call ensime#au_buf_leave(expand("<afile>"))
-    autocmd CursorHold *.java,*.scala call ensime#au_cursor_hold(expand("<afile>"))
-    autocmd CursorMoved *.java,*.scala call ensime#au_cursor_moved(expand("<afile>"))
+    if !has('timers'):
```

Comment:
Ok @Kazy, I've been asking to the nvim guys what to do about it ([start of chat](https://gitter.im/neovim/neovim?at=58cf100a7b3f37e75414e44b)) and we should be ok not defining auto commands when in nvim since `timers` have been around for an year now and people already switched to nvim like new stuff that's for sure :)
So let's remove cursorhold and friends from nvim as it is not acceptable to have both mechanisms running at the same time.
What you reckon? I can test it for a while after you change it.

Cheers.